source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
 
-* 15 Total_Item_Investigations being reported in the DR_D1 Standard View.
+**Option 3**: Platform has multiple databases but the user can only search the whole platform.
 
-The auditor must carry out 15 **outside** tests.
+The auditor MUST run 100 searches, resulting in 100 Searches_Automated (1 per search).
 
-This must result in:
+**Option 4**: Platform has multiple databases but the user can only search a single database at a time.
 
-* 30 Total_Item_Investigations being reported in the DR_D1 Standard View.
+The auditor MUST run 100 searches across at least 10% of the available databases, restricted to one database at a time, resulting in 100 Searches_Regular (1 per search).
 
+**Option 5**: Platform has only one database.
 
-E.2.2.3 Standard View: DR_D2
-''''''''''''''''''''''''''''
+The auditor MUST run 50 searches, resulting in 50 Searches_Regular (1 per search).
 
-Databases Access Denied: This Standard View reports on access-denied activity for databases where a user is denied access because simultaneous user licenses are exceeded or the institution does not have a license for the database.
 
-An audit of this Standard View and related tests requires the following:
+E.2.3.2 Total_Item_Requests
+'''''''''''''''''''''''''''
 
-* The auditor must allow at least 31 seconds between each request unless otherwise specified.
-* Each time a user is denied access, the auditor will record the database on which the denial was produced.
-* Audit tests D2-1 and D2-2 must take place in separate accounts so that each audit test can be separately reported.
-* For each applicable audit test to achieve an audit pass, all metrics reported by the content provider in DR_D2 Standard View from the auditor’s test account must be within a -8% and +3% reliability window of the same audit metrics on the auditor’s report.
+Where possible, 50% of items requested SHOULD be via browsing the platform and 50% via searching the platform. If either browsing or searching is not possible, all items can be requested via the only available option.
 
+The auditor MUST make a total of 80 requests against different items, resulting in 80 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests.
 
-Audit Test D2-1: Limit_Exceeded
-...............................
+Where a platform has fewer than 80 items, the auditor MUST make at least one request per item and testing should result in 80 each of Total_Item_Investigations and Total_Item_Requests, and the item-count of Unique_Item_Investigations and Unique_Item_Requests.
 
-**IMPORTANT NOTE**: This test can only be carried out if the content provider has a concurrent/simultaneous user limit. Any exclusion of tests must be confirmed by COUNTER prior to testing and the auditor notified.
 
-The auditor will perform audit tests based on the relevant option detailed below.
+E.2.3.3 Total_Item_Investigations
```

Comment:
One some platforms it might be impossible to have a Request without a previous Investigation (independent from Request). I assume on those platforms the tests from this section and the previous section would be run together?

Reply:
in most cases the auditors are going to operate in this way, yes